src/Microsoft.Identity.Client/ClientApplicationBase.WithBuilders.cs

```diff
@@ -106,6 +107,32 @@ internal ClientApplicationBase(ApplicationConfiguration config)
             return await handler.RunAsync(CancellationToken.None).ConfigureAwait(false);
         }
 
+        async Task<Uri> IClientApplicationBaseExecutor.ExecuteAsync(
+            AcquireTokenCommonParameters commonParameters,
+            GetAuthorizationRequestUrlParameters authorizationRequestUrlParameters,
+            CancellationToken cancellationToken)
+        {
+            var requestParameters = CreateRequestParameters(commonParameters, UserTokenCacheInternal);
+            requestParameters.Account = authorizationRequestUrlParameters.Account;
+            requestParameters.LoginHint = authorizationRequestUrlParameters.LoginHint;
+
+            if (!string.IsNullOrWhiteSpace(authorizationRequestUrlParameters.RedirectUri))
+            {
```

Comment:
will this fix the issue w/the redirect uri?

Reply:
Do you mean constructor parameter?  what issue are you referring to?

---
In reply to: [250857156](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/833#discussion_r250857156) [](ancestors = 250857156)